app/views/layouts/_header.html.erb

```diff
@@ -30,6 +30,9 @@
     <div class="site-name">
       <%= link_to "Security Education Companion", "/" %>
     </div>
+    <div class="site-name-mobile">
```

Comment:
I think as of #134 @vbrown608 had the mobile header matching the mockups. I don't know if those changed just FYI.

Reply:
This is in addition to the changes she made, which didn't work on smaller mobile screens (i.e. iPhone5).